recipes/dbus/dbus.yaml

```diff
@@ -0,0 +1,61 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+   PKG_DESCRIPTION: "D-Bus is a simple system for interprocess communication and coordination."
+   PKG_LICENSE: "AFL-2.1"
+   PKG_VERSION: "1.15.8"
+
+depends:
+   - libs::expat-dev
+   - libs::glib-dev
+   - use: []
+     depends:
+      - libs::expat-tgt
+      - libs::glib-tgt
+
+checkoutSCM:
+    scm: url
+    url: https://dbus.freedesktop.org/releases/dbus/dbus-${PKG_VERSION}.tar.xz
+    digestSHA1: 73538c86a0b3da9a1c054383543da652eeea122e
+    stripComponents: 1
+
+checkoutDeterministic: True
+checkoutScript: |
+   autoconfReconfigure
+
+buildVars: [DEBUG_DBUS]
```

Comment:
Could we call this `CONFIG_DBUS_DEBUG`? I think having a common schema like `CONFIG_${PKG_NAME}_*` would be beneficial as the number of such options grows...